ydb/public/lib/ydb_cli/commands/ydb_sql.cpp

```diff
@@ -121,6 +132,10 @@ int TCommandSql::RunCommand(TConfig& config) {
     if (ExplainMode) {
         // Execute explain request for the query
         settings.ExecMode(NQuery::EExecMode::Explain);
+    } else if (ExplainAst) {
+        // Execute explain request with full stats for the ast
+        settings.ExecMode(NQuery::EExecMode::Explain);
+        settings.StatsMode(ParseQueryStatsModeOrThrow(CollectStatsMode, NQuery::EStatsMode::Full));
```

Comment:
You don't need stats collection for getting ast. (Also, stats collection leads to executing a query)
https://github.com/ydb-platform/ydb/blob/afbc266232aa2f58d2c41dd93025b9094490b24b/ydb/core/kqp/ut/olap/kqp_olap_ut.cpp#L2528